backend/br_ops/purge_backups.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+
+echo "This script works only in dev amd production spaces as we take backups only in these spaces"
+echo "It relies on naming conventions used in br_nackup.sh"
+echo "It assumes that the S3 bucket named 'backups' has been bound"
+
+cfspace=$(cf target |tail -1|cut -d':' -f2|xargs)
+echo "Space: ${cfspace}"
+
+if [[ "$cfspace" != "dev" && "$cfspace" != "prodiction" ]]; then
+  echo "Error: Backups may only be purged fromr dev or production"
+  exit 1
+fi
+prefix=$(date '+%Y-%m-%d-%H%M')
+export PATH=/home/vcap/deps/0/apt/usr/lib/postgresql/15/bin:$PATH
+python manage.py fac_s3 backups --ls | awk '{print $1}' | grep "db\-backup" | sort -r > "$prefix"_backup_list.txt
```

Comment:
@danswick, @timoballard, and I think that we should move more of the logic here into the `fac_s3` management command, so that
* that command generates the backup list text file
* that command reads application config from settings.py in order to control the retention policy.

Reply:
I would do this in a later PR. This would involve some logic changes, and I'd rather see the current PR tested in `dev` first.